app/controllers/application_controller.rb

```diff
@@ -1,8 +1,17 @@
 class ApplicationController < ActionController::API
+  include ActionController::MimeResponds
```

Comment:
Yes! The ActionController::API doesn't contain MimeResponds. Well spotted though. :)